blocks/ifus/ifus.css

```diff
@@ -0,0 +1,76 @@
+.ifu {
+    display: flex;
+    flex-flow: row nowrap;
+    justify-content: center;
+    padding: 2rem 0 0;
+    max-width: 1300px;
+    margin: auto;
+}
+
+.ifu-result {
+    width: 100%;
+    padding: 2rem 0 0;
+    max-width: 1300px;
+    margin: auto;
+}
+
+.ifu-result.no-result {
+    visibility: hidden;
+    padding: 0;
+}
+
+@media (max-width: 1024px) {
+    .ifu {
+        flex-direction: column;
+    }
+}
+
+.ifu > div {
+    padding: 15px 20px;
+    width: 100%;
+}
+
+.ifu-result > div > div {
+    padding: 0 1rem 2rem;
+    border-bottom: 1px solid #75787b;
+    overflow-wrap: break-word;
+    display: flex;
+    flex-flow: row nowrap;
+    justify-content: space-between;
+}
+
+@media (max-width: 1024px) {
+    .ifu-result > div > div {
+        flex-direction: column;
+    }
+}
+
+.ifu > .ifu-selection > select {
+    display: block;
+    width: 100%;
+    border: 1px solid #666;
```

Comment:
should probably use local var referencing `--mt-global-color-dove-gray`

Reply:
Done